.github/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+## Overview
+Briefly describe the purpose, goals, and changes or improvements made in this pull request.
+
+## Related Issues
+Specify any related issues or pull requests by linking to them.
+
+## Checklist
+- [ ] Update help
+- [ ] Update changelog
+- [ ] Use `labelFor` (if applicable) - [Issue #943](https://github.com/zaproxy/zaproxy/issues/943)
```

Comment:
Will this cause all PRs to link back to this issue?

Reply:
Hmmm probably. I’ll see if I can come up with an Oracle or OpenJDK reference.

Reply:
[This](https://docs.oracle.com/javase/8/docs/api/javax/swing/JLabel.html#setLabelFor-java.awt.Component-)?

Reply:
Yup that seems good.

Reply:
IMO this should be part of the dev guidelines.

Reply:
Okay, I’d be happy to see this bullet handled that way too 🙂